.github/workflows/initi.yaml

```diff
@@ -0,0 +1,141 @@
+name: CI
+
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  ghr_push:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Prepare
+        id: prep
+        uses: ./.github/actions/docker-image-tag
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.REP_PAT }}
+
+      -  name: Build and push
+         uses: docker/build-push-action@v2
+         with:
+           context: .
+           file: ./Dockerfile
+#          push: ${{ github.event_name != 'pull_request' }}
+           push: true
+           tags: ${{ steps.prep.outputs.tag }}
+           cache-from: type=local,src=/tmp/.buildx-cache
+           cache-to: mode=max,type=local,dest=/tmp/.buildx-cache
+
+
+  test_1:
```

Comment:
This `test_i` steps map directly to the official nbdev test. Can we use the same task/job names
as in gitlab?

* read_notebooks:
* check_notebooks_clean:
* check_diff_lib_nbs:
* un_tests:

Reply:
:boom: